frontends/web/src/locales/de/app.json

```diff
@@ -598,6 +601,28 @@
       "title": "Geräteinformationen"
     },
     "expert": {
+      "bip85": {
+        "description": "Erstelle Unterschlüssel von deiner BitBox Wallet",
+        "disclaimer": "Ich verstehe, wie Unterschlüssel funktionieren und die Sicherheitsaspekte",
+        "how": {
+          "description": "Die Unterschlüssel werden als Mnemonik auf deiner BitBox angezeigt. Du kannst so viele Unterschlüssel erstellen, wie du willst. Um einen Unterschlüssel anzuzeigen, wirst du nach der Kindernummer (z.B. 1., 2., 3. usw.) und der gewünschten Länge der Mnemonik (12, 18 oder 24 Recovery-Wörter) gefragt. Denke daran, dass du die <strong>Kindernummer und die Anzahl der Wörter</strong> behalten musst, um dieselbe Mnemonik später anzeigen zu können.",
+          "title": "Wie funktioniert es?"
+        },
+        "recover": {
+          "description": "Du kannst die Mnemonik für die Unterschlüssel immer wieder anzeigen lassen. Wie erwähnt, musst du die <strong>Kindernummer und die Anzahl der Wörter</strong> merken, um dieselbe Mnemonik anzuzeigen. Importiere niemals deine Haupt-BitBox Recovery-Wörter in eine andere Wallet, verwende beim Importieren in eine andere Wallet nur Unterschlüssel.",
```

Comment:
Die Wiederherstellungswörter der "Child Keys" können jederzeit erneut auf der BitBox angezeigt werden. Hierfür sind der Key-Index und die Wortanzahl erforderlich, um denselben "Child Key" erneut zu generieren. Importiere niemals die Wiederherstellungswörter des Backups deiner BitBox-Hauptwallet in eine andere Wallet. Verwende für diesen Zweck ausschließlich dafür vorgesehene "Child Keys".